cobigen-setup/files/cliExecutionCommands.md

```diff
@@ -0,0 +1,20 @@
+CobiGen CLI is installed inside your devonfw distribution. In order to execute it follow the next steps:
+
+1. Now, we have to go into the devonfw installation directory to execute the commands.
+
+`cd devonfw`{{execute T1}}
+
+2.Execute cobigen or cg and the man page should be printed. Main entry point of the CLI. If no arguments are passed, man page will be printed.
+
+`devon cobigen`{{execute T1}}
+
+`devon`{{execute T1}}
+
+`cobigen`{{execute T1}}

```

Comment:
You should also write a sentence about what each of the commands does. The first installs CobiGen within the ide, the second sets the environment variables for devon and the last displays the main page of cobigen with the commands